### PR TITLE
added simple medical-unit-mssc with only two RequestMapings

### PR DESCRIPTION
**Pull request by wojciechGaudnik**

### CODE REVIEW

File: Medical-Units-MSSC/.mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (weronikawalczak) [inline Medical-Units-MSSC/.mvn/wrapper/MavenWrapperDownloader.java]: Most likely we don't need that, let's check if all works without it and remove everything related to mvnw

File: Medical-Units-MSSC/src/main/java/com/clinics/medicalunits/ui/repositorie/MedicalUnitRepository.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
+public interface MedicalUnitRepository extends JpaRepository<MedicalUnit, Long> {
+
+	@Query("SELECT m FROM medical_unit m")
```
Comment (weronikawalczak) [inline Medical-Units-MSSC/src/main/java/com/clinics/medicalunits/ui/repositorie/MedicalUnitRepository.java]: Is that needed ?

Comment (wojciechGaudnik) [reply Medical-Units-MSSC/src/main/java/com/clinics/medicalunits/ui/repositorie/MedicalUnitRepository.java]: My mistake, I couldn't remember findAll();

File: Medical-Units-MSSC/.mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (wojciechGaudnik) [reply Medical-Units-MSSC/.mvn/wrapper/MavenWrapperDownloader.java]: I will try remove it after finish appointment.

**Pull request merged**